clang/lib/Interpreter/Interpreter.cpp

```diff
@@ -702,10 +723,8 @@ llvm::Error Interpreter::LoadDynamicLibrary(const char *name) {
   if (!EE)
     return EE.takeError();
 
-  auto &DL = EE->getDataLayout();
-
-  if (auto DLSG = llvm::orc::DynamicLibrarySearchGenerator::Load(
-          name, DL.getGlobalPrefix()))
+  if (auto DLSG = llvm::orc::EPCDynamicLibrarySearchGenerator::Load(
```

Comment:
Should we not default to in-process and only if we require out-of-process to set these things specifically for out-of-process?

Reply:
`EPC` is `ExecutorProcessControl` -- using these variants means that the system works regardless of the underlying implementation (`SelfExecutorProcessControl` vs `SimpleRemoteEPC`), whereas using `DynamicLibrarySearchGenerator` _only_ works in the current process.

Eventually I want to get rid of the Non-EPC variants. I might even invert the naming scheme and start calling the old ones "LegacyInProcess<XXX>". :)